kruize-clowdapp.yaml

```diff
@@ -264,3 +268,7 @@ parameters:
   value: "info"
 - name: KRUIZE_PARTITION_INTERVAL
   value: "0 0 * * *" # Run the cronjob every day at midnight. Earlier it was set for 16th day
+- name: KRUIZE_CW_LOG_STREAM
+  value: "kruize-stream"
```

Comment:
@kusumachalasani Please update the stream value to `kruize-recommendations` i.e. the Openshift deployment name, the current one seems a bit generic.